src/controllers/issue-controller.ts

```diff
@@ -11,7 +11,8 @@ export class IssueController {
                 "label": activity.name,
                 "description": "",
                 "detail": "",
-                "fullIssue": activity
+                "fullIssue": this.issue,
```

Comment:
issue property is available inside IssueController Claas, so you shouldn't change this.
There is `"fullIssue"` key due to my laziness - I should've change interfaces.